catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -180,7 +180,15 @@ public Metacard getMetacard(String id)
                         TimeUnit.SECONDS.toMillis(10)), false));
         return resultIterable.stream()
                 .collect(Collectors.toMap(result -> result.getMetacard()
-                        .getId(), Function.identity()));
+                        .getId(), Function.identity(), (old, current) -> {
```

Comment:
❗️ For two reasons, I'd like this merge function extracted. Firstly, there is my general dislike for large blocks of code in the middle of streaming logic because it makes seeing the structure harder; secondly, the merge functions are identical.

Reply:
Extracted, 👍